app/views/activity_notification/notifications/default/_default.html.erb

```diff
@@ -0,0 +1,47 @@
+<% content_for :notification_content, flush: true do %>
+  <% notification = NotificationDecorator.new notification %>
+  <div>
+    <div class="uk-grid-collapse uk-grid">
+      <div>
+        <%= notification.sender_profile_picture(class: 'uk-comment-avatar profile-picture') %>
+      </div>
+      <div>
+
+        <h4 class="uk-comment-title uk-margin-remove">
+          <%= notification.sender %>
+        </h4>
+
+        <div class="uk-comment-meta uk-text-break">
+          <%= notification.created_at.strftime("%b %d %H:%M") %>
+          <%= notification.open_link(class: 'uk-link-muted') %>
+          <%= notification.mark_as_read_link(class: 'uk-link-muted') %>
+        </div>
+      </div>
+    </div>
+    <div class="uk-text-break">
+
+      notified you of
```

Comment:
Can you translate these text to Hungarian?

Reply:
I've left this like this, because the default notifications are not used and I feel it easier to have an original template if someone would like to change